src/tests/functional/plugin/conformance/test_runner/op_conformance_runner/include/read_ir_test/read_ir_tests.hpp

```diff
@@ -12,6 +12,10 @@ TEST_P(ReadIRTest, ReadIR) {
     run();
 }
 
+TEST_P(ReadIRTest, QueryModel) {
```

Comment:
we can move this test bosy from shared lib to executable file directly

Reply:
removed